admin/taxonomy/class-taxonomy.php

```diff
@@ -196,13 +198,89 @@ public function localize_term_scraper_script() {
 			// Todo: a column needs to be added on the termpages to add a filter for the keyword, so this can be used in the focus kw doubles.
 			'search_url'                    => admin_url( 'edit-tags.php?taxonomy=' . $term->taxonomy . '&seo_kw_filter={keyword}' ),
 			'post_edit_url'                 => admin_url( 'edit-tags.php?action=edit&taxonomy=' . $term->taxonomy . '&tag_ID={id}' ),
-			'sep'                           => WPSEO_Utils::get_title_separator(),
-			'sitename'                      => WPSEO_Utils::get_site_name(),
+			'title_template'                => WPSEO_Taxonomy::get_title_template( $term ),
+			'metadesc_template'             => WPSEO_Taxonomy::get_metadesc_template( $term ),
 			'contentTab'                    => __( 'Content:', 'wordpress-seo' ),
 		);
 	}
 
 	/**
+	 * Retrieves the title template.
+	 *
+	 * @param object $term taxonomy term.
+	 *
+	 * @return string
+	 */
+	public static function get_title_template( $term ) {
+		$options = get_option( 'wpseo_titles' );
+
+		$title_template = '';
+		if ( isset( $options[ 'title-tax-' . $term->taxonomy ] ) && $options[ 'title-tax-' . $term->taxonomy ] !== '' ) {
```

Comment:
Probably you can wrap this like you did for the posts. The only thing is you don't know the type of `term`. I would check this like:

`if ( is_object( $term ) && property_exists( $term, 'taxonomy'  ) ) {  `